### DISCUSSION TITLE
Incorrect behavior of the plus button in Vivaldi browser.

### DISCUSSION THREAD

**Discussion by sychevivan** [Able-Bear-Software/tab-nodes-tree#94] [Q&A]
In Vivaldi when I open a new tab by Ctrl+T, it opens with the address:
`chrome://vivaldi-webui/startpage?section=Speed-dials`
When I open a new tab using the plus button (introduced in 1.5.0), it opens with the address:
`chrome://newtab`

And for some reason the page address is directly written:
![image](https://github.com/user-attachments/assets/e82df817-9f0b-43cc-9e86-58dec0cbdf00)

Is it possible to make the plus button open the same page that is opened by Ctrl+T (or by the plus button of standard built-in tabs)?

**Comment by Tab-Nodes-Tree** [comment]
@sychevivan Is that URL Vivaldi specific thing? I never use Vivaldi before...

I could add a setting for user to customize the new tab URL. It is not difficult to do, but since this extension is mainly for Chrome though. It is likely to be low priority.

#94

**Comment by sychevivan** [comment]
Apparently this is the default address in Vivaldi because I haven't configured it.
I don't know if it's right to add a new tab URL setting, because I haven't seen such a setting in any browser or extension before. 
Unexpectedly I found the extension “New Tab Redirect”, in which I set the address I need and now the plus button opens the same page as Ctrl+T.
Anyway thanks for the reply!
